Sources/General/ZLEditImageConfiguration.swift

```diff
@@ -190,6 +190,9 @@ public class ZLEditImageConfiguration: NSObject {
     
     /// Whether to support redo in graffiti and mosaic tools. Defaults to false
     public var canRedo = false
+    
+    /// Minimum zoom scale, allowing the user to make the edited photo smaller, so it does not overlap top and bottom tools menu. Defaults to 1.0
+    public var minimumZoomScale = 1.0
```

Comment:
Need to add a chaining-call method for this property.

Reply:
Added, thanks